extensions/authentication-jwt/package.json

```diff
@@ -21,27 +21,29 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@loopback/authentication": "^6.0.1",
-    "@loopback/core": "^2.9.5",
-    "@loopback/rest": "^6.2.0"
+    "@loopback/authentication": "^7.0.0",
+    "@loopback/authorization": "^0.7.0",
+    "@loopback/core": "^2.10.0",
+    "@loopback/rest": "^7.0.0"
```

Comment:
The master already has such versions - https://github.com/strongloop/loopback-next/blob/master/extensions/authentication-jwt/package.json. Please rebase your branch against latest master:

```sh
git remote add upstream git@github.com:strongloop/loopback-next.git
git fetch upstream
git rebase upstream:master
git push -f
```
```